rules/aip0158/response_plural_first_field.go

```diff
@@ -37,13 +41,15 @@ var responsePluralFirstField = &lint.MessageRule{
 		want := utils.GetResourcePlural(utils.GetResource(firstField.GetMessageType()))
 		if want != "" {
 			want = strcase.SnakeCase(want)
+			message = fmt.Sprintf("The name of the first field of a paginated response should be the snake_case plural of the resource, which is %q based on the `(google.api.resource).plural` annotation.", want)
```

Comment:
```suggestion
			message = "Paginated resource response field name should be the snake_case form of the resource type plural defined in the `(google.api.resource)` annotation.",
```

Let's let the `Suggestion` carry the `want` rather than the message itself, since we have one that we can actually "Apply fix" for. Then, just tighten up the language a bit to keep it more concise. WDYT?

This eliminates the `fmt` import to btw.